ms/applink.c

```diff
@@ -132,6 +132,15 @@ OPENSSL_Applink(void)
     return OPENSSL_ApplinkTable;
 }
 
+extern __declspec(dllexport)
+void 
+# if defined(__BORLANDC__)
+__stdcall
+# else
+__cdecl
+# endif
+OPENSSL_SetApplink(void** (*custom)());
```

Comment:
Well, if you consider `__BORLANDC__` commentary at OPENSSL_Applink declaration, you should notice that it's only a coincidence that it was possible to use it. With precondition that declared function doesn't take any arguments, when `__stdcall` and `__cdecl` are binary equivalent and differ only be name decoration. This can't be said about suggested function and therefore one can't play the trick here.